README.md

```diff
@@ -1,26 +1,46 @@
 # token-supply
+
 REST API for CoinGecko and CoinMarketCap
 
-## Run the server locally
+## Setup
 
 Install the dependencies:
 ```
+yarn global add vercel
 yarn install
 ```
 
-Add variables to your local development environment:
+Create the project:
+```
+vercel
+```
+
+Add variables to your environment:
 ```
 cp .env.sample .env
 ```
 
-Build:
+## Deploy the serverless functions locally
+
+Local deployment:
 ```
-yarn build
+vercel dev
```

Comment:
@tnrdd I'm having problems deploying the server on localhost, getting `Transaction reverted without a reason string`:

```
yarn install
cp .env.sample .env
vercel dev
```
```
Vercel CLI 28.0.1
> Creating initial build
Running "yarn run build"
yarn run v1.22.19
$ tsc
✨  Done in 3.59s.
> Success! Build completed
> Ready! Available at http://localhost:3000
========= NOTICE =========
Request-Rate Exceeded  (this message will not be repeated)

The default API keys for each service are provided as a highly-throttled,
community resource for low-traffic projects and early prototyping.

While your application will continue to function, we highly recommended
signing up for your own API keys to improve performance, increase your
request rate/limit and enable other perks, such as metrics and advanced APIs.

For more details: https://docs.ethers.io/api-keys/
==========================
Unhandled rejection: Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0x333A4823466879eeF910A04D473505da62142069","data":"0x18160ddd","accessList":null}, code=CALL_EXCEPTION, version=providers/5.6.8)
```

Did 
I do something wrong in the installation steps?

Reply:
The installation steps are right, seems like it got rate limited. Make sure the api keys fields in the `.env` are filled, otherwise it uses the shared ethers.js api key with a lower rate limit.

Reply:
Yep @aahna-ashina, actually they are already deployed:
https://token-supply.api.nation3.org/api/total-supply
https://token-supply.api.nation3.org/api/circulating-supply